article/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+from django.contrib.auth.models import User
+import uuid
+# Create your models here.
+
+class Article(models.Model):
+    id = models.UUIDField(default=uuid.uuid4,  unique=True, primary_key=True, editable=False)
```

Comment:
primary_key= True means unique and not null True,
id = models.UUIDField(default=uuid.uuid4, primary_key=True, editable=False)

Reply:
Just for more safety purpose